sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -135,6 +144,11 @@ export default defineComponent({
     const isLastPage = ref(false);
     const showNav = ref(false);
     let formInstance: FormIOForm;
+    const searchApplicationsForm = ref({} as VForm);
+    const { checkOnlyDigitsRule, checkNullOrEmptyRule } = useRules();
+    const applicationNumberLabel = "Application number";
```

Comment:
What is the benefit of having const for the label? as they are statis IMO we do not require const.

Reply:
It's used for the label and validations:
![image](https://user-images.githubusercontent.com/78114138/198163839-26897b4d-2a39-4629-8002-78cbd5b214ca.png)

Reply:
Ok. As per our conversation, I removed the consts and let the messages more generic.